Symfony/CS/Test/AccessibleObject.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Test;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class AccessibleObject
+{
+    private $object;
+    private $reflection;
+
+    public static function create($object)
+    {
+        return new self($object);
+    }
+
+    public function __construct($object)
```

Comment:
Can't the constructor be `private`? I see no use of this constructor as is..dunno..